test/specs/constants.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ * We store the salesforcedx-vscode specific constants here. 
+ */
+
+// the name of repos
+export enum repoKeywords {
```

Comment:
I still want to define the enum like this, since we are going to need the name for folder path.